### PR TITLE
E2E Test: fix image comparison tests & add windows plot reference images

### PR DESCRIPTION
**Pull request by testlabauto**
Fix the logic for attaching screenshots in a way they can be retrieved post run from CI and update plots tests to have master images for windows runs

### QA Notes

All smoke tests should pass

### CODE REVIEW

**Review by midleman — APPROVED** [review]
I know you are waiting for @jonvanausdeln approval, but @midleman approves too! ✅

**Review by jonvanausdeln — APPROVED** [review]
LGTM!

**Pull request merged**
